ui/app/templates/vault/cluster/access/oidc/keys/index.hbs

```diff
@@ -21,7 +21,7 @@
       <div class="level-left">
         <div>
           <Icon @name="key" class="has-text-grey-light" />
-          <span class="has-text-weight-semibold is-underline">
+          <span class="has-text-weight-semibold is-underline" data-test-item>
```

Comment:
selectors have to be more specific because `innerText` on linked block dom elements now include the `Interactive` text